src/sv-pipeline/scripts/aggregate_picard.py

```diff
@@ -0,0 +1,276 @@
+def read_in(file_name, label):
+    started = False
+    newlist = []
+    wgsfile = file_name
+    
+    with tfio.gfile.GFile(file_name, "r") as inp:
+        for line in inp:
+            intable = line.startswith(label)
+            if intable == True:
+                started = True
+                continue
+            if started:
+                if line != '\n':
+                    newlist.append(line.rstrip('\n'))
+            if started and line == '\n':
+                return newlist
+                break
+                
+def newtable(file_name, label):
+    columnnames = read_in(file_name, label)[0].split("\t")
+    values = read_in(file_name, label)[1].split("\t")
+    dataframe = pd.DataFrame(columnnames).T
+    dataframe.columns = columnnames
+    dataframe_length = len(dataframe)
+    dataframe.loc[dataframe_length] = values
+    dataframe = dataframe.drop(0)
+    return dataframe
+
+def addrows(file_name, label):
+    onerow = newtable(file_name, label)
+    for i in range(len(read_in(file_name, label))):
+        nextrow = read_in(file_name, label)[i].split("\t")
+        onerow.loc[i] = nextrow
+    onerow = onerow.drop(0)
+    return onerow
+
+def concatrows(table, suffix):
+    newdata = pd.DataFrame()
+    for row in range(1, len(table) + 1):
+        addrow = pd.DataFrame(table.loc[row]).T.reset_index(drop=True)
+        addrow = addrow.add_suffix(suffix[row-1])
+        newdata = pd.concat([newdata, addrow], axis = 1)
```

Comment:
Dataframe concatenations as well as creation are expensive operations and should be avoided if possible. You should be able to rewrite this function to avoid these operations. Feel free to reach out if you need any help in this.